.gitignore

```diff
@@ -104,4 +105,6 @@ venv.bak/
 .mypy_cache/
 
 # asdf version manager settings
-.tool-versions
\ No newline at end of file
+.tool-versions
+
+
```

Comment:
Too many blank lines!

You might be able to configure your editor so that it fixes these on save - look for something about 'unix line endings'

Reply:
Adding this line to your preferences should make it work:
`"default_line_ending": "unix"`